### ISSUE TITLE
508 defect 1, Launchblocker! screen readers and keyboard users, Financial Status Report, List Loop Component is not keyboard accessible

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#35747]
- [508-defect-1 :exclamation: Launchblocker]

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1

Issue Title: Debt Resolution, Financial Status Report, List Loop Component is not keyboard accessible.
Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

<h2>User Story or Problem Statement:</h2>
<p>As a screen reader user who navigates exclusively with the keyboard, I expect all components to be properly labeled and accessible.</p>
<h2>Details</h2>
In the list loop component which comes up multiple times throughout this form:
<ul>
<li>The edit-combo box has an ARIA label "Autosuggest input." This overrides the form label and causes a screen reader to only say "Autosuggest input" when focus is moved to it.</li>
<li>The link to add an item is not in the tab order and is not keyboard accessible. The user can arrow to it, but cannot activate it. In some instances it is not labeled.</li>
<li>The link to edit an item is not labeled and in some instances is not in the tab order. </li>
</ul>
<h2>Acceptance Criteria</h2>
<ul>
<li>Screen readers identify the edit-combo according to its purpose.</li>
<li>Links to add and edit items are properly labeled.</li>
<li>Links to add and edit items are in the tab order.</li>
</ul>
<h2>Environment</h2>
<p>JAWS with Chrome and NVDA with Chrome on a PC running Windows 10.</p>
<h2>Steps to Recreate</h2>
<ul>
<li>Sign into Staging with user+1.</li>
<li>Open the menu whose label is the user's first name and choose "My VA."</li>
<li>Choose Manage Your VA Debt.</li>
<li>Choose Request Help with VA Debt.</li>
<li>Choose "Request Help with VA Form 5655.</li>
<li>Navigate through the form until you are asked to add payroll deductions. This is the first of many occurances of the list loop. </li>
</ul>
<h2>Proposed Solution (if known)</h2>
<ul>
<li>Delete the ARIA label from the edit-combo box so that the form element label is read.</li>
<li>Set the subindex of the add item link to 0 and ensure that it has propper link text.</li>
<li>Ensure that the edit item link has proper link text and that the subindex is zero.</li>
</ul>

**Comment by taylorkaren** [comment]
Info from @micahchiang: The Forms Library Team could look at removing the autosuggest-input aria label, but I’m unclear on the second 2 points of the proposed solution because I can tab to and activate the add item link without any problems.
